### DISCUSSION TITLE
Can I deploy  a second redis cluster in the same k8s?

### DISCUSSION THREAD

**Discussion by doronl** [PayU/redis-operator#143] [General]
@voltbit 
Can I deploy  a second redis cluster in the same k8s? do I need a different namespace for that? 2 operators, one in each namespace?

**Comment by voltbit** [comment]
For now the operator can handle a single cluster per namespace. So yes you would need two different deployments of the operator. I dont think we tested the operator with more than one redis cluster per K8s cluster but as long as you use them in different namespaces I expect them to work.

If not, let us know and open an issue detailing your experience.

Unrelated to the question, version 2.0 was released this week.

**Reply by doronl** [reply]
Thanks, Yes I have seem v2.0 was released, I will continue my testing with v1.1 and if\when I have I release that satisfies my minimal requirements I will try 2.0... already made few changes to the code base so I may run the POC so far
